series/views.py

```diff
@@ -41,8 +43,39 @@ class ShowDetailsView(View):
 
     def get(self, request, id: int):
         show = retrieve_show(id)
+        if request.user.is_authenticated:
+            sub = APIShow.objects.filter(pk=id, followers__id=request.user.pk).exists()
+        else:
+            sub = False
```

Comment:
In Django, the general philosophy is "fat model, thin views". Now that we have an `APIShowManager`, perhaps we could place this logic under the manager to make it reusable?

Reply:
Fixed it for you 😄